.github/workflows/release-docker-nightly.yml

```diff
@@ -360,29 +360,29 @@ jobs:
   #       with:
   #         github-token: ${{ secrets.ACCESS_TOKEN }}
 
-  #     - name: Update titan-op-node Kubernetes resources
```

Comment:
please uncomment if we need

Reply:
yes, I follow @rlgns98kr to uncomment these lines. We will uncomment if needed

Steven's PR: https://github.com/tokamak-network/tokamak-thanos/pull/125

Reply:
Ok. We will make k8s resources of thanos-sepolia-nightly and update this CI.